packages/perseus/src/widgets/__tests__/radio/choice_test.jsx

```diff
@@ -0,0 +1,201 @@
+// @flow
```

Comment:
I'm surprised that there weren't any tests for this before.  **praise:** thanks for adding these. 💚